content/tutorials/get-started/set-up-secure-signing.md

```diff
@@ -0,0 +1,134 @@
+
+To use this configuration:
+
+1. [Install `rippled`](install-rippled.html) on the chosen machine.
+
+    Be sure that this machine meets the minimum [system requirements for `rippled`](system-requirements.html).
+
+2. Configure the `rippled` machine to have a static IP address in the local private LAN, or use a private DNS server to give the server a consistent name.
+
+    With IPv4, private LAN addresses commonly start with `10.`, `192.168.`, or `172.(16 to 31).`. With IPv6, private LAN addresses typically start with `fc` or `fd`.
+
+3. Configure the `rippled` machine to accept connections from your local private network.
+
+    In the `rippled`'s config file, modify the `[port_rpc_admin_local]` and `[port_ws_admin_local]` stanzas to use your server's private-LAN address:
+
+        [port_rpc_admin_local]
+      	port = 5005
+      	ip = 10.1.0.0 # Changed from 127.0.0.1
+      	admin = 10.1.1.2 # Only the client with this IP is treated as admin
+      	protocol = http
+
+        [port_ws_admin_local]
+      	port = 6006
+      	ip = 10.1.0.0 # Changed from 127.0.0.1
+      	admin = 10.1.1.2 # Only the client with this IP is treated as admin
+      	protocol = ws
+
+4. If your `rippled` machine runs a software firewall, configure the firewall to accept connections on your local private network at the JSON-RPC and Websocket ports you configured in the previous step. (Ports `5005` and `6006` in the previous example.)
+
+    The exact configuration depends on your firewall software.
+
+5. On the machine(s) that will submit transactions, connect to your server using your `rippled` server's private IP address and the configured ports. Use the [sign method][] (for single signatures) or [sign_for method][] (for multi-signatures).
+
+    The example configuration from the previous steps uses port `5005` for JSON-RPC connections and port `6006` for Websocket connections.
+
+6. Keep the `rippled` server running, updated, and in sync with the network while you're using it.
+
+    **Note:** You _can_ turn off your `rippled` server when you're not sending transactions, but it can take up to 15 minutes to sync with the network when you start it up again.
+
+
+## Use a Client Library with Local Signing
+
+[![Diagram of using a client library with local signing](img/secure-signing-client-library.png)](img/secure-signing-client-library.png)
+
+This configuration uses a client library in the programming language you are using to sign transactions locally. You need a client library for the programming language you use. Ripple publishes the following client libraries that can sign transactions for the XRP Ledger locally:
+
+- **RippleAPI (ripple-lib) for JavaScript**
+    - [Setup](get-started-with-rippleapi-for-javascript.html)
+    - [API Reference](rippleapi-reference.html)
+- **ripple-libpp for C++**
+    - [Source and Documentation](https://github.com/ripple/ripple-libpp/)
```

Comment:
I don't think we should recommend ripple-libpp, instead:  https://github.com/ripple/rippled/tree/develop/Builds/linux#signing-library

Reply:
for that matter, I don't know why we don't just make one or two simple command line utils that sign transactions using this lib and include them with rippled...but we don't for now.

Reply:
We should make such tools, period. Let's do so for **1.3**.

Reply:
Would be nice if they worked in a way that doesn't expose the secret to the process listing and shell history, btw.